envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/EnvoyDefaultFilters.kt

```diff
@@ -63,7 +64,9 @@ class EnvoyDefaultFilters(
     val defaultAuthorizationHeaderFilter = { _: Group, _: GlobalSnapshot ->
         authorizationHeaderToMetadataFilter()
     }
-    val defaultEgressFilters = listOf(defaultHeaderToMetadataFilter, defaultEnvoyRouterHttpFilter)
+    val defaultEgressFilters = listOf(
+        defaultHeaderToMetadataFilter, defaultServiceTagFilter, defaultEnvoyRouterHttpFilter
```

Comment:
It will be better to add any feautre flag to disable/enable this feature or to control if we want to log or reject this kind of requests

Reply:
OK. Are you OK if I add this flag to use by lua script as an "if" around `rejectServiceTagDuplicatingAutoServiceTag` function, instead of disabling the whole lua script? 
Because the script `egress_service_tags.lua` will be doing more in the near feature, so I would prefer to not disable it completely (it would need to be changed in the next PR)

Reply:
yes, it is ok